pkg/asset/machines/azure/machines.go

```diff
@@ -188,6 +188,9 @@ func provider(platform *azure.Platform, mpool *azure.MachinePool, osImage string
 
 	if platform.CloudName == azure.StackCloud {
 		spec.AvailabilitySet = fmt.Sprintf("%s-cluster", clusterID)
+
+		// Public Azure has moved to use image galleries, but ASH is still using managed images.
+		spec.Image.ResourceID = fmt.Sprintf("/resourceGroups/%s/providers/Microsoft.Compute/images/%s", rg, clusterID)
```

Comment:
ouch. my bad for missing this. is there an equivalent terraform that still creates managed images for ASH or do we need to add that as well?

Reply:
ah...i see azurestack has separate tf files with the managed image...